projects/ExtraRepository.cmake

```diff
@@ -0,0 +1,33 @@
+foreach(extra_repository ${EXTRA_REPOSITORIES})
```

Comment:
Note that I intentionally left this out as I consider the original approach a hack and would rather include a bunch of additional .cmake files for external projects. Originally I figured you could just drop them into cura-build/projects/ however it might be nicer to add a cmake variable that points to a directory that contains extra files so those can be separately managed.